mhkit/loads/extreme.py

```diff
@@ -618,15 +620,21 @@ def mler_coefficients(rao, wave_spectrum, response_desired):
     ----------
     rao: numpy ndarray
         Response amplitude operator.
-    wave_spectrum: pd.DataFrame
-        Wave spectral density [m^2/Hz] indexed by frequency [Hz].
+    wave_spectrum: pandas Series, pandas DataFrame, xarray DataArray, or xarray Dataset
+        Wave spectral density [m^2/Hz] indexed by frequency [Hz]. 
+        DataFrame and Dataset inputs should only have one data variable
     response_desired: int or float
         Desired response, units should correspond to a motion RAO or
         units of force for a force RAO.
+    dimension: string (optional)
+        Name of the xarray dimension corresponding to time.
+        If not supplied, time is assumed to be the first dimension.
```

Comment:
If this is always time then I think "time_dimension" is a more descriptive name and  increases the clarity for the user to the kwargs intended use without reading the docstring.

I think the docstring should explicitly state that this is only used in xarray (it is certainty implied). So I am thinking along the lines of _~"time_dimesion has no impact on pandas type inputs"

Reply:
Got it. That makes sense and is a good idea. Thanks for clarifying; I misunderstand in our last discussion. I'll rename the "dimension" input here and in the other relevant functions